src/VisualStudio/Core/Impl/SolutionExplorer/SourceGeneratedFileItems/SourceGeneratedFileItemSource.cs

```diff
@@ -156,7 +156,7 @@ public void BeforeExpand()
             lock (_gate)
             {
                 // We should not have an existing computation active
-                Contract.ThrowIfNull(_cancellationTokenSource == null);
+                Contract.ThrowIfTrue(_cancellationTokenSource is not null);
```

Comment:
```suggestion
                Contract.ThrowIfNull(_cancellationTokenSource);
```

Reply:
Backwards: we want to assert that it *is* null, not that it's not null. (I made this same mistake myself.)